hie.yaml

```diff
@@ -117,6 +117,9 @@ cradle:
     - path: "libs/cardano-ledger-core/src"
       component: "lib:cardano-ledger-core"
 
+    - path: "libs/cardano-ledger-core/test-lib"
```

Comment:
I think the folder was renamed to `testlib` as well, no?

Reply:
Good catch!!! Thank you!

Reply:
Fixed